tests/common.sh.in

```diff
@@ -201,4 +227,81 @@ onError() {
 
 trap onError ERR
 
+isTestOnSystemNix() {
+  [[ "${TEST_SYSTEM_NIX:-}" = 1 ]]
+}
+
```

Comment:
These functions should be made a bit more accurate, and "integrate" better with the existing helper functions, which should also be made a bit more accurate or precise.